### ISSUE TITLE
Reduce / remove API data downtime during updates

### ISSUE THREAD

**Issue by chorsley** [cybergreen-net/pm#58]
When we load data into RDS, there's currently a 20 minute site outage while the data is loaded.

This should be performed as a near-atomic operation (e.g. loaded into temp tables first), so there is very little downtime.

**Comment by rufuspollock** [comment]
@chorsley i'd suggest that we load into a new RDS instance and switch the frontend to a new database instance. That also allows for fallback if something breaks.

**Comment by aaronkaplan** [comment]
okay. when and how? And what's exactly needed to test this? (note: that's part of atomatic SoW). @rgrp please propose an actionable solution which we can test in the next weeks.

**Comment by rufuspollock** [comment]
@aaronkaplan i would say this is a dependency on the orchestrator setup so this will be blocked on that. Also this issue was moved to https://github.com/cybergreen-net/infrastructure/issues/6 - let's update there.

**Comment by rufuspollock** [comment]
DUPLICATE of https://github.com/cybergreen-net/infrastructure/issues/6
